app/components/TxHistory/TxHistoryRow/RegularTxRow.js

```diff
@@ -47,8 +53,11 @@ const RegularTxRow = ({
           { txDirection === "transfer" ?
             <>
               <T id="txHistory.from" m="From " />
-              { txAccountName }
-              { accountMessage }
+              { txAccountNameDebited }
+              <T id="txHistory.account" m=" Account" />
+              <T id="txHistory.to" m=" To " />
+              { txAccountNameCredited }
+              <T id="txHistory.account" m=" Account" />
```

Comment:
Much nicer!

For i18n purposes, it's better to use a single parametrized message (so instead of individual `<T>` use a single `<T m="From {srcAccount} to {dstAccount}"`). I believe the `" Account"` suffix is redudant.

Reply:
Agree. I also changed it for tx in and out as well.